tests/Helpers/Tokens/IdTokenVerifierTest.php

```diff
@@ -75,8 +75,11 @@ public function testThatTokenWithInvalidIssuerFails()
 
     public function testThatTokenWithMissingSubFails()
     {
-        $verifier  = new IdTokenVerifier('__test_iss__', uniqid(), new SymmetricVerifier('__test_secret__'));
-        $builder   = (new Builder())->issuedBy('__test_iss__');
+        $verifier  = new IdTokenVerifier('__test_iss__', '__test_aud__', new SymmetricVerifier('__test_secret__'));
+        $builder   = (new Builder())
```

Comment:
Tests changed because the standard JWT claims are now verified before the ID-token specific ones.